api/src/main/resources/rdr.yaml

```diff
@@ -0,0 +1,202 @@
+#This file provide the end points for interaction with RDR
+
+
+swagger: '2.0'
```

Comment:
Since this is a standalone, short API spec, it might be a good first experiment in OpenApi 3...